digiroad2-oracle/src/main/resources/db/migration/V0_14__add_placeholder_text.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE PROJECT_ACTION_HISTORY (
```

Comment:
Pitäisikö olla primary key? Käyttötapaus ei sellaista tunnu tarvitsevan

Reply:
Hmmm.
Kuvittelisi tosiaan, ettei tuohon tarttis kauheasti muualta viitata.
Jos suorat kantahaut onnistuvat ilman sitäkin, niin en mä näe erityistä tarvetta.
Ja jos tarttis manuaalipäivittää ton rivejä joskus, niin siinä DBeaver rutisisi primary keyn puuttumisesta. Mut lokiriveihin ei yleensä kai ole tarkoitus koskea...
En siis keksi minäkään, milloin tuolle olisi tarvetta.

Sen  jälkeen primary keyn määrittämistä jää puoltamaan enää yleinen "for completeness' sake", enkä nyt oikein tiedä, onko se riittävän hyvä syy.
Toisaalta, jos primary keyn määrittäminen ei vie paljoa resursseja, niin ei kai sen olemassa oleminen haittaakaan. 🤷